cdap-docs/reference-manual/source/http-restful-api/workflow.rst

```diff
@@ -133,4 +135,307 @@ Examples
      - ``{"key1":[{"nodeName": "MyExitNode", "value":"1"}]}``
    * - Description
      - Retrieves the value for the key *key1* at node *MyExitNode* from the token in the scope *User* for a specific run of *MyWorkflow*
- 
\ No newline at end of file
+
+
+Workflow Statistics
+===================
+These requests provide statistics across *successful* runs of a Workflow, in a time
+interval, for a succession of runs, or between two specific runs. These requests will help
+in detecting which jobs might be responsible for delays or problems.
+
+Statistics of Successful Runs
+-----------------------------
+
+This request returns general statistics about all *successful* workflow runs in a particular time interval, 
+with an analysis based on a series of (optionally) provided percentiles::
+
+  GET <base-url>/namespaces/<namespace>/apps/<app-id>/workflows/<workflow-id>/statistics?
+    start=<start-time>&end=<end-time>&percentile=<percentile-1>&percentile=<percentile-2>...
+    
+where
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<namespace>``
+     - Namespace ID
+   * - ``<app-id>``
+     - Name of the application
+   * - ``<workflow-id>``
+     - Name of the workflow
+   * - ``<start-time>``
+     - Start time of runs; default is ``now`` (optional)
```

Comment:
Looking at the code we can also pass timestamp and if we do it should be in seconds
https://github.com/caskdata/cdap/blob/develop/cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/WorkflowStatsSLAHttpHandler.java#L91-L100